src/components/schema/ExistFilter.tsx

```diff
@@ -0,0 +1,56 @@
+import AutocompletedField from 'components/shared/toolbar/AutocompletedField';
+import { useIntl } from 'react-intl';
+import { AutoCompleteOption, FieldFilter } from './types';
+
+interface Props {
+    fieldFilter: FieldFilter;
+    setFieldFilter: (value: FieldFilter) => void;
+    disabled?: boolean;
+}
+
+function ExistFilter({ fieldFilter, setFieldFilter, disabled }: Props) {
+    const intl = useIntl();
+
+    const fieldOptions: AutoCompleteOption[] = [
+        {
+            id: 'all',
+            label: intl.formatMessage({
+                id: 'schemaEditor.table.filter.option.all',
+            }),
+        },
+        {
+            id: 'must',
+            label: intl.formatMessage({
+                id: 'schemaEditor.table.filter.option.must',
+            }),
+        },
+        {
+            id: 'may',
+            label: intl.formatMessage({
+                id: 'schemaEditor.table.filter.option.may',
+            }),
+        },
+    ];
+
+    return (
+        <AutocompletedField
+            label={intl.formatMessage({
+                id: 'schemaEditor.table.filter.label',
+            })}
+            options={fieldOptions}
+            defaultValue={fieldOptions.find(
```

Comment:
Changing the default value on a controlled `Autocomplete` component will result in an error reported in the console; the `value` property should be set in this case. We discussed this earlier today, but I wanted to leave a comment for record.

Reply:
done